qa/cephfs/mount/kclient/overrides/distro/stock/ubuntu_22.04.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/ubuntu_22.04.yaml
```

Comment:
why not rhel9?

Reply:
Sure. I will add that, but yeh I think we don't have rhel9 for reef.